docs/_sass/_main.scss

```diff
@@ -2,6 +2,16 @@
 @import 'colors';
 @import 'fonts';
 
+$color-appBG: $color-green-appBG;
+$color-highlightBG: $color-green-highlightBG;
+$color-borders: $color-green-borders;
+$color-icons: $color-green-icons;
+$color-blue: $color-blue200;
+$color-text: $color-white;
+$color-link: $color-link;
+$color-link-hovered: $color-link-hovered;
+$color-green: $color-green400;
```

Comment:
It might make more sense to use a semantic name here too, like `$color-success: $color-green400`

Then anytime we have a success button or something else, we use the success variable and not the color value